skimage/transform/tests/test_polar.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+from skimage.draw import circle
+from skimage.transform import cart2pol
+from numpy.testing import assert_equal
```

Comment:
I agree with you in general, but for this particular case, the transform is not supposed to change the value of the pixels, just to reorganize them. Thus, to me, it looks safer to make sure that there is no modification at all of pixel values.